stories/topics/con-azure-disaster-recovery.adoc

```diff
@@ -0,0 +1,17 @@
+[id="con-azure-disaster-recovery_{context}"]
+
+= Disaster Recovery
+
+When you deploy {AAPonAzureNameShort}, you must enable or disable Disaster Recovery in the `Business Continuity` tab of the form.
+There is no default setting for Disaster Recovery.
+
+If you want to enable Disaster Recovery on an existing instance of {AAPonAzureNameShort}, contact Red Hat customer support.
```

Comment:
Just wondering here if we should hyperlink "Red Hat customer support" directly to support, or is this fairly common knowledge?

Reply:
I thought it better not to link, as different customers might have different contact points with Red Hat.